src/components/SpruceForm/Widgets/DateTimePicker.tsx

```diff
@@ -20,10 +20,12 @@ export const DateTimePicker: React.VFC<
     ? utcToZonedTime(new Date(value), timezone)
```

Comment:
value is initially null which throws this warning

![image](https://user-images.githubusercontent.com/4605522/195205792-73ca1bb5-c907-484b-9113-08978edaf171.png)

![image](https://user-images.githubusercontent.com/4605522/195205821-71612601-1a2e-4509-b17c-eab493bfd339.png)
Its probably easiest to set a default value for it until we load the data